src/presign/participant.rs

```diff
@@ -836,15 +840,6 @@ impl PresignParticipant {
         &self,
         identifier: Identifier,
     ) -> Result<Vec<crate::round_three::Public>> {
-        if !has_collected_all_of_others(
-            &self.other_participant_ids,
-            &self.storage,
-            StorableType::PresignRoundThreePublic,
-            identifier,
-        )? {
-            return bail!("Not ready to get other participants round three publics just yet!");
-        }
-
         let mut ret_vec = vec![];
```

Comment:
How come the check here is no longer needed?

Reply:
The idea was that if `has_collected_all_of_others()` fails then `storage.retrieve() ` will fail for one of the participants and you'll get the same result.